app/models/developer.rb

```diff
@@ -85,4 +86,12 @@ def missing_fields?
   def send_admin_notification
     NewDeveloperProfileNotification.with(developer: self).deliver_later(User.admin)
   end
+
+  def changed_to_invisible?
+    search_status_previously_changed? to: "invisible"
```

Comment:
```suggestion
    search_status_previously_changed?(to: "invisible")
```

Also, TIL about this method!

Reply:
I forget when they changed dirty tracking (we're still on the only methods at work) but I do remember they changed it so I had to look it up too 😅